mozilla/foundation.mozilla.org
# updated all variables to have whitespace in between brackets

# Description
Related PRs/issues: #10095 


This PR updates all html templates that had djlint `t001` errors.

I first used Tibor's suggested regex expression: `sed -i.bak "s|{{\([A-z0-9._]*\)}}|{{ \1 }}|g" **/*.html` which did a great job at reducing the number of errors. 

However, there were still some other `t001` errors caused by formatting such as `{% block%}`. So I went through the list of remaining errors and updated them myself. 



# Screenshots:



### Linting output before:
<img width="500" alt="Screenshot 2023-02-08 at 4 01 17 PM" src="https://user-images.githubusercontent.com/18314510/217686747-326b5641-14d3-451e-8da2-29d260ce3d11.png">


### Linting output after:
<img width="700" alt="Screenshot 2023-02-08 at 4 33 22 PM" src="https://user-images.githubusercontent.com/18314510/217686782-5ed9c51d-4af4-4308-a342-58a5f4ea9a01.png">




# Steps to test

1. Check out this branch
2. On line 9 of `pyproject.toml`, paste: 
```
ignore="D004,D018,H005,H006,H007,H008,H008,H010,H011,H012,H013,H014,H015,H016,H017,H019,H020,H021,H022,H023,H024,H025,H026,H029,H030,H031,H033,J004,J018,T002,T003,T027,T028,T032,T034,H017"
```
3. Comment out / remove lines 15-250
4. In the terminal run the command `inv djlint-lint`
5. If the linter returns 0 errors, testing is complete! 
6. Undo changes made to `pyproject.toml`
